SwiftSyntaxDevUtils/Sources/swift-syntax-dev-utils/commands/Build.swift

```diff
@@ -0,0 +1,27 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import ArgumentParser
+import Foundation
+
+struct Build: ParsableCommand, Builder {
+  @OptionGroup
+  var arguments: BuilderArguments
+
+  func run() throws {
+    try buildProduct(productName: "SwiftSyntax-all")
+
+    // Build examples
+    try buildExample(exampleName: "AddOneToIntegerLiterals")
+    try buildExample(exampleName: "CodeGenerationUsingSwiftSyntaxBuilder")
```

Comment:
Since https://github.com/apple/swift-syntax/pull/1660 you can build `Examples-all` to build all examples.